pom.xml

```diff
@@ -223,6 +224,11 @@
       </exclusions>
     </dependency>
     <dependency>
+      <groupId>com.atlassian.fugue</groupId>
+      <artifactId>fugue</artifactId>
+      <version>${fugue.version}</version>
+    </dependency>
```

Comment:
Only considering Maven, yes. But I am not sure if it will be embedeed in the hpi / jpi if we do so ...